test/04-signature.js

```diff
@@ -10,5 +10,17 @@ tape('04 signature type', function (t) {
   t.deepEqual(encoded[0].slice(0, 2), Buffer.from([4, 0]), 'detected signature')
 
   t.deepEqual(bfe.decode(encoded), values, 'properly decoded')
+
+  t.doesNotThrow(
+    () => bfe.encode('ZG9n.sig.bob'),
+    'unknown signature suffix means encoded as a string'
+  )
```

Comment:
This is an interested edge case - the lack of sigil means we can't be sure if this is Signature or a String!
I was tempted to write a regex which was something like: `/^<base64pattern>\.sig.\.\w+$/`

But that would be a breaking change and we currently only have one signature and this seems unlikely needed atm